src/static/custom-button.ts

```diff
@@ -20,4 +22,17 @@ window.addEventListener("load", function () {
 
     const scheme_pan = document.getElementsByClassName('scheme-container')[0];
     scheme_pan.children[0].appendChild(auth_pan);
-});
\ No newline at end of file
+
+    fetchUserInfo(login_button, logout_button);
+});
+
+async function fetchUserInfo(login_button: HTMLButtonElement, logout_button: HTMLButtonElement) {
+    const res = await fetch(`${window.location.origin}/user`);
```

Comment:
And here as well, `/user` to `/auth/user`

Reply:
Done! ✅